style.css

```diff
@@ -224,6 +224,7 @@ button:hover {
 
 #team {
     position: relative;
+    overflow: hidden;
```

Comment:
Comment: Setting overflow to 'hidden' ensures that the video remains within the section boundaries.